packages/main/src/plugin/container-registry.ts

```diff
@@ -1040,6 +1041,38 @@ export class ContainerProviderRegistry {
     }
   }
 
+  // Find all containers that match the against the given project name to
+  // the label com.docker.compose.project
+  // and then restart all the containers that match that label
+  async restartContainersByProject(engineId: string, projectName: string): Promise<void> {
```

Comment:
wondering if it could not be more generic (like you can provide the searched label)

like if I want to restart all containers matching a different label than `com.docker.compose.project` ?